server/app/views/applicant/ApplicantProgramSummaryView.java

```diff
@@ -204,23 +204,37 @@ private DivTag renderQuestionSummary(
 
     if (data.isAnswered() || haveAnswerText) {
       final ContainerTag answerContent;
-      if (data.encodedFileKey().isPresent()) {
+
+      if (!data.encodedFileKeys().isEmpty()) {
+        answerContent = div();
+        for (int i = 0; i < data.encodedFileKeys().size(); i++) {
+          if (i > 0) {
+            answerContent.with(br());
```

Comment:
Rather than replacing `br` with `p`, the idea is to remove the if-statement and then wrap the anchor tag in a paragraph, so that there's naturally a break after each line.  On line 218, it would look like this:  
```
answerContent.with(
  p(
     a(fileName).withHref(fileLink).withClass(BaseStyles.LINK_TEXT)
   )
);
```
As for the other use below, where we replace new lines with the `br`, I think this would be more complicated to change since we have to respect the new lines in the answer.  I'm sure it can be done, but it might not be worth the effort.

Reply:
Right, thank you! Thinking more along those lines, this is a list, so I'll use an <ol> + <li>s.